sdk/web-pubsub/web-pubsub-express/src/cloudEventsDispatcher.ts

```diff
@@ -241,7 +241,7 @@ export class CloudEventsDispatcher {
 
     // check if hub matches
     const hub = utils.getHttpHeader(request, "ce-hub");
-    if (hub !== this.hub) {
+    if (hub?.toLowerCase() !== this.hub.toLowerCase()) {
```

Comment:
Can we normalize to uppercase instead? https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2015/code-quality/ca1308-normalize-strings-to-uppercase

Alternatively, perhaps we can rely on the platform for this support, since I don't know if hub names can contain diacritics or not?
```suggestion
    if (hub?.localeCompare(this.hub, undefined, {sensitivity: "accent"}) === 0) {
```

@vicancy curious about your thoughts here

Reply:
Hub name needs to match `^[A-Za-z][A-Za-z0-9_`,.[\\]]{0,127}$`, so hub names does not contain diacritics